sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/Segment.cs

```diff
@@ -63,7 +63,7 @@ public virtual SegmentCursor GetCursor()
             return new SegmentCursor(
                 segmentPath: ManifestPath,
                 shardCursors: shardCursors,
-                currentShardPath: _shards[_shardIndex].ShardPath);
+                currentShardPath: _shards.Count > 0 ? _shards[_shardIndex].ShardPath : null);
```

Comment:
Could you add a test where we attempt to deserialize and use cursor with `currentShardPath` = `null` ?
We should make sure we can handle such cursor.

Reply:
we should also make sure other languages can consume such cursor. (and possibly check if they have similar bug..)

Reply:
@gapra-msft @xiafu-msft, can you see if Java and Python have this bug?

Reply:
We can definitely handle `currentShardPath = null`: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/SegmentFactory.cs#L82

Reply:
great, so do we already have test for that edge case ?

Reply:
Done.